kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -85,3 +85,33 @@ $ scaleCluster.sh -d domain1 -n weblogic-domain-1 -c cluster-1 -r 3
 domain.weblogic.oracle/domain1 patched
 [2021-02-26T19:04:14.466 UTC][INFO] Successfully patched replicas for cluster 'cluster-1'!
 ```
+
+### Script to view the status of any WebLogic cluster
+
+The `clusterStatus.sh` script can be used to view the status of any WebLogic cluster in the WebLogic domain managed by WKO. The status contains information about the minimum, maximum, goal, current, and ready replica count for a WebLogic cluster. This script dumps a table containing the status for WebLogic cluster(s) in one or more domains across one or more namespaces. See the script `usage` information by using the `-h` option.
+
+Use the following command to view the status of all WebLogic clusters in all domains across all namespaces.
+```
+$ clusterStatus.sh
+
+WKO Cluster Status -n "" -d "" -c "":
+
+namespace          domain            cluster    min  max  goal  current  ready
+---------          ------            -------    ---  ---  ----  -------  -----
+ns-kvmt            mii-domain1       cluster-1  1    5    5     5        5
+weblogic-domain-1  domain1           cluster-1  0    4    2     2        2
+weblogic-domain-1  domain1           cluster-2  0    4    0     NA       NA
```

Comment:
We are showing NA here because the operator did not report a value for the given attribute. But it looks like we can safely interpret NA is '0' in this case, which would be far more intuitive IMO.  Assuming NA always can be safely interpreted means 0, should we either (a) correct the operator code to report a value for this field, or (b) modify the script to report '0' instead of NA when an attribute is missing, or (c) do both?  Or maybe NA truly means 'don't know'?

@rjeberhard Would you like to weigh in?

Reply:
I'm not very sure but since the replica count goal is 0, displaying NA (not applicable) for current and ready fields might be self-explanatory. Displaying 0 for current/ready might mean that pods are not yet created/ready.

Reply:
The goal is '0' - so if displaying 0 for current/ready "might mean that pods are not yet created/ready", why is that a bad thing?

If it turns out that we can safely substitute 0 for NA, not only would it be more intuitive, but also any script that uses this script as input wouldn't have to add special case handling for the NA string...

Reply:
I'm not saying it's a bad thing, I just thought that displaying NA might be OK.

Reply:
Done.